project/sites/ircommission/themes/ircommission_theme/src/scss/components.scss

```diff
@@ -51,6 +51,10 @@
         flex-direction: column;
         position: relative;
       }
+
+      h2 {
+        color: $brand-tertiary;
```

Comment:
Should the colour be #111 or #141f71?
It looks like you have change the headings colour but still referenced $brand-tertiary in the scss file.
If t is #141f71 then its good though.

Reply:
The front page block headings should still be #141f71 as far as I know. The rest of the headings should be #111.